docs/en_us/platform_api/source/shared/courses_response_values.rst

```diff
@@ -0,0 +1,223 @@
+.. _Course:
+
+Course
+*******
+
+* count (integer): The number of courses in the catalog.
+* next (string): The URL for the next page of results.
+* previous (string): The URL for the previous page of results.
+* results (array): See :ref:`Results`.
+
+.. _Results:
+
+Results
+*******
+
+Parent: ``course``
```

Comment:
Yes, `results` here is an array of course objects.  This is a pretty standard response to deal with anything we plan on paginating.  If you wanted to get technical (and I don't) you could say that all collection endpoints return the same single `collection object` which consists of count, next, previous, and a results array.  But we typically skip over that part and talk about the endpoint as being a collection of  `course objects` and the count, next, etc, is just the boilerplate pagination cursor metadata that comes with it.